src/inpage/webpack.config.js

```diff
@@ -12,15 +12,15 @@ const config = {
   module: {
     rules: [
       {
-        test: /\.(js|jsx)$/u,
-        exclude: /node_modules/u,
```

Comment:
oof. this is unideal. I guess we can kick this can down the road though

Reply:
also works if we do:
```
include: [
          path.resolve(__dirname, '..'),
          path.resolve(__dirname, '..', '..', 'node_modules', '@metamask'),
          path.resolve(__dirname, '..', '..', 'node_modules', 'superstruct'),
        ],
```
but...

Reply:
🙃